Core/NoOpRunner.Core/PowerUpsContainer.cs

```diff
@@ -13,23 +13,36 @@ namespace NoOpRunner.Core
     /// </summary>
     public class PowerUpsContainer : ShapesContainer, IObserver
     {
+        private readonly Random random = new Random(DateTime.Now.Millisecond); 
```

Comment:
We have the RandomNumber singleton, could use that

Reply:
Yes